src/ddb/tests/ddb_smoke_tests.sh

```diff
@@ -68,8 +70,5 @@ run_cmd diff /tmp/ddb_new_value /tmp/ddb_value_dump
 
 rm -f /tmp/ddb_commands
 touch /tmp/ddb_commands
-echo "ls" >> /tmp/ddb_commands
-echo "ls [0]" >> /tmp/ddb_commands
-echo "ls [0]/[0]" >> /tmp/ddb_commands
-echo "dump_superblock" >> /tmp/ddb_commands
+{ echo "ls"; echo "ls [0]"; echo "ls [0]/[0]"; echo "dump_superblock"; } >> /tmp/ddb_commands
```

Comment:
Heh, this is kind of nifty, hadn't encountered this construction before. It's probably fine, but might be a little more maintainable to use a heredoc, e.g.:
```
cat <<EOF>/tmp/ddb_commands
ls
ls [0]
ls [0]/[0]
dump_superblock
EOF
```

Reply:
Agreed, this looks better. I don't seem to be able to run shellcheck locally, I'll push and see if shellcheck is happy with it.